radixdlt-core/radixdlt/src/integration/java/com/radixdlt/integration/targeted/recovery/RecoveryLivenessTest.java

```diff
@@ -160,34 +156,54 @@ public RecoveryLivenessTest2(int numNodes, long epochCeilingView) {
     }
   }
 
-  @Category(Slow.class)
-  @RunWith(Parameterized.class)
   public static class RecoveryLivenessTest3 extends RecoveryLivenessTest {
 
     @Parameters
     public static Collection<Object[]> numNodes() {
-      return List.of(new Object[][] {{3, 88L}, {4, 88L}});
+      return List.of(new Object[][] {{2, 88L}});
     }
 
     public RecoveryLivenessTest3(int numNodes, long epochCeilingView) {
       super(numNodes, epochCeilingView);
     }
   }
 
-  @Category(Slow.class)
-  @RunWith(Parameterized.class)
   public static class RecoveryLivenessTest4 extends RecoveryLivenessTest {
 
     @Parameters
     public static Collection<Object[]> numNodes() {
-      return List.of(new Object[][] {{2, 1L}, {10, 100L}});
+      return List.of(new Object[][] {{4, 88L}});
     }
 
     public RecoveryLivenessTest4(int numNodes, long epochCeilingView) {
       super(numNodes, epochCeilingView);
     }
   }
 
+  public static class RecoveryLivenessTest5 extends RecoveryLivenessTest {
+
+    @Parameters
+    public static Collection<Object[]> numNodes() {
+      return List.of(new Object[][] {{2, 1L}});
+    }
+
+    public RecoveryLivenessTest5(int numNodes, long epochCeilingView) {
+      super(numNodes, epochCeilingView);
+    }
+  }
+
+  public static class RecoveryLivenessTest6 extends RecoveryLivenessTest {
+
+    @Parameters
+    public static Collection<Object[]> numNodes() {
+      return List.of(new Object[][] {{10, 100L}});
+    }
+
+    public RecoveryLivenessTest6(int numNodes, long epochCeilingView) {
+      super(numNodes, epochCeilingView);
+    }
+  }
+
```

Comment:
I tried it, but seems it increased the execution time. Did you have better results?

Reply:
Yup on my local machine